index.html

```diff
@@ -317,7 +317,38 @@ <h1>カフェで読書もコワーキングも。快適のフロアスペース
     </section>
   </main>
 
-  <footer></footer>
+  <footer>
+    <div class="info">
+      <a class="logo" href="#">
+        <img src="./images/logo_foot.png">
+      </a>
+      <p class="info-det">〒532-0011　大阪市淀川区西中島6丁目8-23テスト
+        <br>TEL.06-1234-5678 FAX.06-1234-5678　営業時間 / 9:00 - 22:00　定休日 / 不定休
+      </p>
```

Comment:
閉じタグの方とそろえるため改行しましょう。
こういう風に
```suggestion
      <p class="info-det">
        〒532-0011　大阪市淀川区西中島6丁目8-23テスト
        <br>TEL.06-1234-5678 FAX.06-1234-5678　営業時間 / 9:00 - 22:00　定休日 / 不定休
      </p>
```